deps/spidershim/src/v8isolate.h

```diff
@@ -51,6 +51,8 @@ struct Isolate::Impl {
   mozilla::Maybe<internal::RootStore> eternals;
   std::vector<MessageCallback> messageListeners;
   void* embeddedData[internal::kNumIsolateDataSlots];
+  mozilla::Atomic<bool> serviceInterrupt;
+  mozilla::Atomic<bool> terminatingExecution;
```

Comment:
I was following some spider monkey code that does this, thinking that we'd need to support workers/threads in the future. Now looking into it, it seems electron does not support workers.